crates/runtime/src/container.rs

```diff
@@ -535,10 +535,14 @@ mod test {
             }]
         );
 
-        #[cfg(target_os = "linux")]
-        assert_eq!(container.mapped_host_ports, super::BTreeMap::new());
-        #[cfg(not(target_os = "linux"))]
-        assert_ne!(container.mapped_host_ports, super::BTreeMap::new());
+        assert_eq!(
```

Comment:
What's this about? Seems like we used to assert that `mapped_host_ports` was empty on linux, and now we'd assert that it's non-empty regardless of OS. Why?

Reply:
It was broken by the recent change to make publish_ports a thing. This test doesn't typically run in CI because `flowctl` isn't built, but does run locally.

This used to correspond to the behavior of port mapping when we used target_os.